packages/network/src/chart/Network.jsx

```diff
@@ -40,6 +40,7 @@ export const propTypes = {
   waitingForLayoutLabel: PropTypes.string,
   width: PropTypes.number.isRequired,
   layout: PropTypes.object,
+  keepAspectRatio: PropTypes.bool,
```

Comment:
could you update the readme with a description of what this does? I also like `preserveAspectRatio` over `keep` but nbd.